src/transformers/models/vit_mae/modeling_tf_vit_mae.py

```diff
@@ -1235,16 +1334,19 @@ def call(
             output_hidden_states=output_hidden_states,
             return_dict=return_dict,
             training=training,
+            interpolate_pos_encoding=interpolate_pos_encoding,
         )
 
         latent = outputs.last_hidden_state
         ids_restore = outputs.ids_restore
         mask = outputs.mask
 
-        decoder_outputs = self.decoder(latent, ids_restore)  # [batch_size, num_patches, patch_size**2*3]
+        decoder_outputs = self.decoder(
+            latent, ids_restore, interpolate_pos_encoding=interpolate_pos_encoding, pixel_values=pixel_values
+        )  # [batch_size, num_patches, patch_size**2*3]
```

Comment:
nit - convention is to put comments above the relevant line 

```suggestion
        # [batch_size, num_patches, patch_size**2*3]
        decoder_outputs = self.decoder(
            latent, ids_restore, interpolate_pos_encoding=interpolate_pos_encoding, pixel_values=pixel_values
        )
```